File: lista-de-exercicios/exercicio-11/Ordenador.test.ts
import { jest } from '@jest/globals';
import { Ordenador } from './Ordenador';

const ordenador = new Ordenador(
    [586, 480, 964, 843, 894, 954, 533, 221, 526, 237]);

describe('Classe Ordenador', () => {
    test('Deve retornar o array ordenado de forma crescente', () => {
        expect(ordenador.ordenar()).toStrictEqual(
            [221, 237, 480, 526, 533, 586, 843, 894, 954, 964]);
    })
})

File: lista-de-exercicios-java/src/test/java/org/example/exercicio11/OrdenadorTest.java
package org.example.exercicio11;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class OrdenadorTest {
    @DisplayName("Deve retornar o array ordenado de forma crescente")
    @Test
    void deveRetornarArrayEmOrdemCrescente() {
        Ordenador ordenador = new Ordenador(new int[] {586, 480, 964, 843, 894, 954, 533, 221, 526, 237});

        assertArrayEquals(new int[]{221, 237, 480, 526, 533, 586, 843, 894, 954, 964}, ordenador.ordenar());
    }
}